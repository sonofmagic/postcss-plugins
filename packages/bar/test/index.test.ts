import postcss from 'postcss'
import myPlugin from '@/index'

describe('index', () => {
  it('foo bar', () => {
    const { css } = postcss([
      myPlugin(),
    ]).process(`page,:host {}`)
    expect(css).toMatchSnapshot()
  })
})
