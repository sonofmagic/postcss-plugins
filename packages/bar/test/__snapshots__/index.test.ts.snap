// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`index > foo bar 1`] = `"page,:host,wx-root-content {}"`;
